buf/registry/module/v1/download_service.proto

```diff
@@ -48,7 +48,10 @@ message DownloadRequest {
     // Specific file types to request.
     //
     // If not set, all file types are returned.
-    repeated FileType file_types = 2 [(buf.validate.field).repeated.items.enum.defined_only = true];
+    repeated FileType file_types = 2 [
+      (buf.validate.field).repeated.items.enum.defined_only = true,
+      (buf.validate.field).repeated.items.enum.not_in = 0
```

Comment:
```suggestion
      (buf.validate.field).repeated.items.enum.not_in = [0]
```

Reply:
For repeated fields being set _not_ in a message literal, you use this singular format (and repeat the same line to add more), and do not wrap it in `[]`